jerry-core/ecma/operations/ecma-objects.c

```diff
@@ -1362,7 +1362,7 @@ ecma_op_object_get_property_names (ecma_object_t *obj_p, /**< object */
   const bool obj_is_builtin = ecma_get_object_is_builtin (obj_p);
 
   const size_t bitmap_row_size = sizeof (uint32_t) * JERRY_BITSINBYTE;
-  uint32_t names_hashes_bitmap[ECMA_OBJECT_HASH_BITMAP_SIZE / bitmap_row_size];
+  JERRY_ALLOCA(names_hashes_bitmap, uint32_t, ECMA_OBJECT_HASH_BITMAP_SIZE / bitmap_row_size);
 
   memset (names_hashes_bitmap, 0, sizeof (names_hashes_bitmap));
```

Comment:
Cppcheck complains about this memset and rightfully so. Previously `names_hashes_bitmap` was guaranteed to be a VLA, for which sizeof gave meaningful result. But for MSVC, it becomes a pointer, so this memset will only clear the first some bytes of the alloca'd array, as sizeof will give the size of the pointer, not the size of the allocated array. `ECMA_OBJECT_HASH_BITMAP_SIZE / bitmap_row_size * sizeof (uint32_t)` will always work, even if its much uglier than before.